tests/integration/basic_example_it/basic_example.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <ydb-cpp-sdk/client/driver/driver.h>
+#include <ydb-cpp-sdk/client/table/table.h>
+
+#include <gtest/gtest.h>
+
+#include <vector>
+
+using namespace NYdb;
+using namespace NYdb::NTable;
+
+struct RunArgs {
```

Comment:
По кодстайлу правильно:
```suggestion
struct TRunArgs {
```